redfish_client/resource.py

```diff
@@ -112,6 +112,13 @@ def find_object(self, key):
                     return result
 
     def execute_action(self, action_name, payload):
+        """
+        Perform an action supported by the resource.
+
+        Args:
+          action: The field representing the action to perform.
```

Comment:
There is no `action` parameter. This should be replaced by the docstring for `action_name` argument.

Reply:
👍